deployments/infrastructure.nix

```diff
@@ -63,6 +51,11 @@ in {
       groups.live-production = {};
     };
 
+    deployment.keys.tarsnap = {
```

Comment:
This should be set only in production

Reply:
Done

Reply:
(I didn't expect us to not want to backup the current, non-`mainnet` deployer..)